book_source/03_intermediate_users_guide/04_remote_execution.Rmd

```diff
@@ -266,20 +273,20 @@ are additional entries to add to the job.sh file generated to
 run the model. This can be used to make sure modules are loaded
 on the HPC cluster before running the actual model.
 
-## Running PEcAn code for modules remotely
+## Running PEcAn code for remotely
 
-You can compile and install the model specific code pieces of
+You do not need to download PEcAn fully on your remote machine. You can compile and install the model specific code pieces of
 PEcAn on the cluster easily without having to install the
 full code base of PEcAn (and all OS dependencies). All of the
 code pieces depend on PEcAn.utils to install this you can
-run the following on the cluster:
+run the following on the remote machine:
 
 ```
 devtools::install_github("pecanproject/pecan", subdir = 'utils')
 ```
 
 Next we need to install the model specific pieces, this is done
-almost the same:
+almost the same (example for ED2):
 
 ```
 devtools::install_github("pecanproject/pecan", subdir = 'models/ed')
```

Comment:
Github isn't letting me comment since it's 'grey' but a few lines down from this line is a whole section of text that's specific to geo.bu.edu. Is this really the correct place to document our internal, machine-dependent configurations?

Reply:
I left this in as an example of how you would do it so people could see an applied example.